### PR TITLE
Fix and improve SDK 6 doc about tests

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: SDK6UserGuide/testProject.rst [hunk 1]
```diff
@@ -212,6 +212,11 @@ This can be convenient to quickly execute one test for example, without requirin
 Refer to the Gradle `Test filtering <https://docs.gradle.org/current/userguide/java_testing.html#test_filtering>`__
 documentation for more details on how to filter the tests and on the available patterns.
 
+.. warning::
+
+   At the moment, only class-level filtering is supported. 
+   This means that, for instance, you are unable to run a single test method within a test class.
```
Comment (gjadi-mej) [inline SDK6UserGuide/testProject.rst:218]:
```suggestion
   This means that, for instance, it is not possible to run a single test method within a test class.
```

**Review by gjadi-mej — APPROVED** [review]
Approved, apply the suggestion if you're ok with it.

File: SDK6UserGuide/testProject.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/testProject.rst:218]: So you don't like what ChatGPT suggests? ;) It is because of the usage of "you"?

Comment (gjadi-mej) [reply SDK6UserGuide/testProject.rst:218]: the `you` thing, yes. Not a formal rule, I know.

Comment (tdelhomenie-microej) [reply SDK6UserGuide/testProject.rst]: Applied. Not sure too.

**Pull request merged**